article/sections/_7_latent_confounding.tex

```diff
@@ -1,5 +1,224 @@
+models 2 and 3 to the true confounder. The main findings of this exercise are the following:
+
+
+Using the true variables believed to be confounded (i.e. method 3 where only 
+travel time and cost are used to recover the confounder) leads to a better 
+recovery of the true confounder. Figure \ref{fig:qq-plot-method-2} and \ref{fig:qq-plot-method-3} show a QQ plot of the true 
+and recovered confounders under models 2 and 3 respectively. Looking at those 
+figures, one can see that the distribution of the recovered substitute 
+confounder under method 3 is closer to that of the true confounder than method 
+2. This suggests that it may be better to run the deconfounder algorithm based 
+on a hypothesized causal graph, rather than just running it on all the 
+observed covariates. Please refer to Section \ref{sec:graph-construction} for how to build plausible causal graphs. 
+
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{qq-plot-method-2.png}
+   \caption{QQ plot of true confounder against recovered confounder using all covariates}
+   \label{fig:qq-plot-method-2}
+\end{figure}
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{qq-plot-method-3.png}
+   \caption{QQ plot of true confounder against recovered confounder using all covariates}
+   \label{fig:qq-plot-method-3}
+\end{figure}
+
+% \begin{figure}
+% \centering
+% \begin{minipage}{.5\textwidth}
+%   \centering
+%   \includegraphics[width=.4\linewidth]{qq-plot-method-2.png}
+%   \captionof{figure}{QQ plot of true confounder against recovered confounder using all covariates}
+%   \label{fig:qq-plot-method-2}
+% \end{minipage}%
+% \begin{minipage}{.5\textwidth}
+%   \centering
+%   \includegraphics[width=.4\linewidth]{qq-plot-method-3.png}
+%   \captionof{figure}{QQ plot of true confounder against recovered confounder using only confounded covariates}
+%   \label{fig:qq-plot-method-3}
+% \end{minipage}
+% \end{figure}
+
+
+Additionally, and perhaps most importantly, the effectiveness of the 
+deconfounder algorithm is very sensitive to small errors and misfits in the 
+recovered confounder. Although method 3 returns a relatively good fit of the 
+true confounder (based on the QQ plot), the adjusted coefficients on travel 
+time and cost do not exhibit any reduction in the bias resulting from 
+ommitting the true confounder, and the coefficients on the recovered 
+confounder are highly insignificant. This raises questions about the 
+usefulness of the deconfounder algorithm in practice. While this limitation 
+may raise questions on the usefulness of the deconfounder algorithm, it is 
+important to point out that it is not only a by-product of the deconfounder 
+algorithm itself. In fact, it is one of the built in characteristic of ommitted
+variable bias, and perhaps more broadly, the problem of error-in-variables in 
```

Comment:
Interesting. I think of error-in-variables as a special case of omitted variable bias, not the other way around. For example, there are types of omitted variable bias that are not errors-in-variables. Latent confounding is great example. I don't know that there are types of errors-in-variables that are not omitted variables though.